test_schemas/en/test_custom_page_titles.json

```diff
@@ -0,0 +1,865 @@
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            "skip_conditions": [
+                                {
+                                    "when": [
+                                        {
+                                            "list": "household",
+                                            "condition": "less than",
+                                            "value": 2
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "title": "Individual Section",
+            "summary": {
+                "show_on_completion": true,
+                "page_title": "Summary for person {list_item_position}"
+            },
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "content": {
+                                "contents": [
+                                    {
+                                        "description": {
+                                            "placeholders": [
+                                                {
+                                                    "placeholder": "person_name",
+                                                    "transforms": [
+                                                        {
+                                                            "arguments": {
+                                                                "delimiter": " ",
+                                                                "list_to_concatenate": {
+                                                                    "identifier": ["first-name", "last-name"],
+                                                                    "source": "answers"
+                                                                }
+                                                            },
+                                                            "transform": "concatenate_list"
+                                                        }
+                                                    ]
+                                                }
+                                            ],
+                                            "text": "In this section, we are going to ask you questions about <strong>{person_name}</strong>."
+                                        }
+                                    },
+                                    {
+                                        "list": ["date of birth"],
+                                        "title": "You will need to know personal details such as"
+                                    }
```

Comment:
This is a bit out of sync with the content of the individual section - It asks for the name again. DOB would probably make more sense.